department-of-veterans-affairs/va.gov-cms
# VACMS-4405: Disable facility api tablefield cells.

## Description
See #4405

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/109047027-63244400-76a3-11eb-81b3-0063981353d9.png)

## QA steps
As an administrator, go to `/node/84/edit` and visually verify that you can't edit the table cells.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
